src/LaravelDebugbar.php

```diff
@@ -470,15 +470,20 @@ public function __construct($transport, $laravelDebugbar)
                         public function send(RawMessage $message, Envelope $envelope = null): ?SentMessage
                         {
                             return $this->laravelDebugbar['time']->measure(
-                                'mail: '. Str::limit($message->getSubject(), 100),
+                                'mail: ' . Str::limit($message->getSubject(), 100),
                                 function () use ($message, $envelope) {
                                     return $this->originalTransport->send($message, $envelope);
                                 },
                                 'mail'
                             );
                         }
-                        protected function doSend(SentMessage $message): void {}
-                        public function __toString(): string{ $this->originalTransport->__toString(); }
+                        protected function doSend(SentMessage $message): void
+                        {
+                        }
+                        public function __toString(): string
+                        {
+                            $this->originalTransport->__toString();
```

Comment:
return missing
```diff
- $this->originalTransport->__toString();
+ return $this->originalTransport->__toString();
```

Reply:
This should be addressed in #1474, where I based this on. I can rebase once that is done and merged.

Also this looks likes an automated fix, so might also need to be reported/fixed in the autofixer.

Reply:
Actually after a second look I noticed this already was incorrect before the auto-styling was applied. Probably might be better to fix this in another PR once #1474 and this have been merged.